### ISSUE TITLE
When download utf8 filename via  Response::sendfile()  , filename is not correct in IE11

### ISSUE THREAD

**Issue by pigochu** [yiisoft/yii2#11953]
Example Code

~~~php
return Yii::$app->getResponse()->sendFile($filepath, $attachment->filename);
~~~

if $attachment->filename is "中文.txt"  <== this is chinese filename with utf8 encoded.

When I download 
- Chome & Firefox save correct filename
- IE 11 is not correct

In http://php.net/manual/en/function.header.php , yjf_victor wrote :
![image](https://cloud.githubusercontent.com/assets/1013744/16832701/0de4414e-49de-11e6-89c4-ac25a1107317.png)

So I change my code looks like:

~~~php
$response = Yii::$app->getResponse()->sendFile($filepath, $attachment->filename);
$response->headers['Content-Disposition'] = 'attachment; '
        . sprintf('filename="%s"; ', rawurlencode($attachment->filename))
        . sprintf("filename*=utf-8''%s", rawurlencode($attachment->filename));
        return $response;
~~~

IE11 & Chrome & Filefox can download file with correct filename.

**Comment by cebe** [comment]
I remember a similar issue report but I could not find it.

**Comment by samdark** [comment]
https://github.com/yiisoft/yii2/issues/10563 ?

**Comment by cebe** [comment]
Yes, thanks.

**Comment by samdark** [comment]
Closed as duplicate of #10563.
